yjit.rb

```diff
@@ -147,8 +147,8 @@ def _print_stats
       return unless stats
 
       $stderr.puts("***YJIT: Printing YJIT statistics on exit***")
-      $stderr.puts("Number of bindings allocated: %d\n" % stats[:binding_allocations])
-      $stderr.puts("Number of locals modified through binding: %d\n" % stats[:binding_set])
+      $stderr.puts("Number of bindings allocated: %d\n" % stats[:binding_allocations].to_i)
+      $stderr.puts("Number of locals modified through binding: %d\n" % stats[:binding_set].to_i)
```

Comment:
Curious why you added all the `.to_i` calls in this file? Are these values not already integers?

Reply:
Right, I reverted it. It was during development, when I only had a partial solution, some of theses keys were missing and the code would raise. The `to_i` helped me figure out what was going on, but it wasn't meant to stick in the final PR.